conf/machine/xilinx-zynqhf.conf

```diff
@@ -19,7 +19,7 @@ UBOOT_ARCH = "arm"
 # we're not building a full u-boot in OE, justfw_{print|set}env and mkimage,
 # thus we use a generic machine.
 #UBOOT_MACHINE ?= "config"
-UBOOT_MACHINE ?= "nicrio9068"
```

Comment:
Why is it necessary to change this variable in particular?

Reply:
the main reason of changing the machine, is because all these while nicrio 9068 always build with 2012 branch. I believe not much validation went through on that. in this case, changing the machine to machine that build with 2017 give us more confident on building with 2017 branch. but underneath, the env tool will not be impacted since it's the tool commonly use for all targets.

Reply:
So there aren't any substantive changes between the `crio9068` and `elvisiii` defconfigs? You're just switching it to the newer defconfig for currency?

Reply:
for the env tool, yes, there is not any substantive changes.

Reply:
OK. That's a satisfactory explanation from my angle, unless @bstreiff or @gratian disagrees.